scripts/bump.sh

```diff
@@ -52,6 +52,8 @@ else
     node scripts/change-readme.js "$RELEASE_CHANNEL"
 fi
 
+    echo "::set-output name=version::$NEXT_EXTENSION_VERSION"
```

Comment:
The indentation looks a bit off. Did you run shell lint?

Reply:
Good catch, no I didn't. Will fix it.